### PR TITLE
feat(authn): clientinfo authn support

### PR DESCRIPTION
**Pull request by ysfscream**
https://emqx.atlassian.net/browse/ED-1605

### CODE REVIEW

**Comment by ysfscream** [comment]
image

File: src/components/ObjectArrayEditor.vue [hunk 1]
```diff
@@ -18,8 +18,10 @@
             <CustomFormItem :prop="getProp($index, key)" :rules="getFormItemRules(key)">
               <SchemaFormItem
                 v-model="displayTableData[$index][key]"
-                :type="(value.type as any)"
-                :symbols="(value.symbols as string[] | number[] | undefined)"
+                :type="value.type"
+                :format="value.format"
+                :symbols="value.symbols"
```
Comment (Kinplemelon) [inline src/components/ObjectArrayEditor.vue:23]: 🙏🙏🙏🙏

Comment (ysfscream) [reply src/components/ObjectArrayEditor.vue:23]: what happened? 😂

**Review by Kinplemelon — APPROVED** [review]

File: src/components/ObjectArrayEditor.vue (hunk 1 above)
Comment (Kinplemelon) [reply src/components/ObjectArrayEditor.vue:23]: You've solved some problems that I was supposed to handle.

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-4287)

**Pull request merged**